arkouda/dtypes.py

```diff
@@ -174,7 +200,11 @@ def __repr__(self) -> str:  # type: ignore
     np.uint64,
     BigInt,
 )
-ARKOUDA_SUPPORTED_DTYPES = frozenset([member.value for _, member in DType.__members__.items()])
+
+# TODO: bring supported data types into parity with all numpy dtypes
+# ARKOUDA_SUPPORTED_DTYPES = frozenset([member.value for _, member in DType.__members__.items()])
+ARKOUDA_SUPPORTED_DTYPES = frozenset(["bool", "float", "float64", "int", "int64",
+                                      "uint", "uint64", "uint8", "bigint", "str"])
```

Comment:
Super minor, but since we've got the types supported right now written manually here, it might be good to put a comment with the list of unsupported types, so it's easier to keep track of those

Reply:
Sounds good, I'll add it.